### PR TITLE
fix: delete combination check

### PR DESCRIPTION
**Pull request by miurahr**
Checking Qt version and modules combination become unfeasible, because release of new versions are frequent.
Also aqtinstall now have a feature to list versions, modules, archives.  

This is a proposal of removal of combination check feature.

- Delete combinations.json and relevant CI script to update it
- Delete feature to print warning with check combinations
- Delete Settings class methods and field to examine combination database
- Remove tests to check combinations
- Update expectation of tests, which expect showing warn.

### CODE REVIEW

**Comment by pzhlkj6612** [comment]
Will close:

- #748 
- #807

.

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ All notable changes to this project will be documented in this file.
 `Unreleased`_
 =============
 
+Deprecated
+----------
+* Qt version/architecture/modules checking (#807, #812)
+
```
Comment (pzhlkj6612) [inline docs/CHANGELOG.rst:19]: Deprecated? I think a deprecated function is still available. We just deleted it, so I suggest using a new word. Which word?

In addition:

```suggestion
Deprecated
----------
* Qt version/architecture/modules checking (#748, #807, #812)

```

.

**Review by pzhlkj6612 — CHANGES_REQUESTED** [review]

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (miurahr) [reply docs/CHANGELOG.rst:19]: I would like to use "Deleted"

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -13,6 +13,10 @@ All notable changes to this project will be documented in this file.
 `Unreleased`_
 =============
 
+Deprecated
+----------
```
Comment (miurahr) [inline docs/CHANGELOG.rst:17]:
```suggestion
Deleted
-------
```

**Review by pzhlkj6612 — APPROVED** [review]
lgtm

**Pull request merged**